server/paymentProviders/paypal/payment.js

```diff
@@ -100,15 +100,15 @@ export async function executePayment(order) {
 
 /** Create transaction in our database to reflect a PayPal charge */
 export async function createTransaction(order, paymentInfo) {
-  /* The `* 100` in the next lines convert from PayPal format in
-     dollars to Open Collective format in cents */
   const transaction = paymentInfo.transactions[0];
-  const amountFromPayPal = parseFloat(transaction.amount.total) * 100;
-  const paypalFee = parseFloat(get(transaction, 'related_resources.0.sale.transaction_fee.value', '0.0')) * 100;
+  const amountFromPayPal = parseFloat(transaction.amount.total);
+  const paypalFee = parseFloat(get(transaction, 'related_resources.0.sale.transaction_fee.value', '0.0'));
+  const amountFromPayPalInCents = Math.trunc(amountFromPayPal * 100);
+  const paypalFeeInCents = Math.trunc(paypalFee * 100);
```

Comment:
@Betree Just curious, why `Math.trunc` vs `Math.round`?

Reply:
The examples I used were rounded above the value. I just made some more research and found out that this is not always the case:

> 0.29 * 100 
28.999999999999996

I'll make a PR with an update, thanks for pointing that out!